recipe/e70878a523ffbc3d9f7d15fc84c2d321515cee61.patch

```diff
@@ -0,0 +1,39 @@
+From e70878a523ffbc3d9f7d15fc84c2d321515cee61 Mon Sep 17 00:00:00 2001
+From: Samuel Debionne <samuel.debionne@esrf.fr>
+Date: Thu, 22 Apr 2021 14:50:34 +0200
+Subject: [PATCH] Add default value for cxx and cxxflags options for the cxx
+ toolset
+
+---
+ src/engine/build.sh | 13 ++++++++++++-
+ 1 file changed, 12 insertions(+), 1 deletion(-)
+
+diff --git a/src/engine/build.sh b/src/engine/build.sh
+index 58f41cb4af..9f655a371d 100755
+--- a/src/engine/build.sh
++++ b/src/engine/build.sh

```

Comment:
the path in the patch must match the location of the file in the boost archive

Reply:
I never get these things right the first time...